### DISCUSSION TITLE
returning array of severe weather alert ID consisting of KEEPALIVE string in one of the weather ID

### DISCUSSION THREAD

**Discussion by ChasingDeveloper** [weather-gov/api#510] [Q&A]
by Passing lat = 39.14 and lon= -77.20 to https://api.weather.gov/alerts/active?point=lat,lon it returns the severe weather alert information. Where as in one of ID returns as - "urn:oid:2.49.0.1.840.0-KEEPALIVE-74" 
 
and using that ID upon requesting to https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0-KEEPALIVE-74, it returns the severe weather information but no headline in it response as below: 
`{
"@context": [
"https://geojson.org/geojson-ld/geojson-context.jsonld",
{}
],
"id": "https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0-KEEPALIVE-74",
"type": "Feature",
"geometry": null,
"properties": {
"@id": "https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0-KEEPALIVE-74",
"@type": "wx:Alert",
"id": "urn:oid:2.49.0.1.840.0-KEEPALIVE-74",
"areaDesc": "Montgomery",
"geocode": {},
"affectedZones": [
"https://api.weather.gov/zones/county/MDC031"
],
"references": [],
"sent": "2022-01-03T15:58:17+00:00",
"effective": "2022-01-03T15:58:17+00:00",
"onset": null,
"expires": "2022-01-03T16:08:17+00:00",
"ends": null,
"status": "Test",
"messageType": "Alert",
"category": "Met",
"severity": "Unknown",
"certainty": "Unknown",
"urgency": "Unknown",
"event": "Test Message",
"sender": "w-nws.webmaster@noaa.gov",
"senderName": "NWS",
"headline": null,
"description": "Monitoring message only. Please disregard.",
"instruction": "Monitoring message only. Please disregard.",
"response": "None",
"parameters": {}
}
}`

can you please let me know why the headline is null for this ID ? Please help me to understand.

**Comment by JJKraw** [comment]
Status is "Test". Description is "Monitoring message only. Please disregard." What more do you need to know?
